view/frontend/web/js/view/payment/method-renderer/tinka.js

```diff
@@ -46,22 +46,35 @@ define(
     ) {
         'use strict';
 
+        $.validator.addMethod('validateAge', function (value) {
```

Comment:
I think 'validateAge' in the future should be added as custom validator to be accessible for all payment methods.